server/schema/order/res.js

```diff
@@ -0,0 +1,63 @@
+import {Order, OrderDetail, Menu, Modifier, sequelize} from '../../models'
+import {_auth} from '../../util'
+
+async function createOrderDetail(orderDetails, orderId) {
+  try {
+    let menuIds = orderDetails.map(orderDetail => orderDetail.menuId)
+    let menus = await Menu.findAll({where: {id: menuIds}})
+    let arrModifierIds = orderDetails.map(orderDetail => orderDetail.modifierIds)
+    let modifierIds = [...new Set([].concat.apply([], arrModifierIds))]
+    let modifiers = await Modifier.findAll({where: {id: modifierIds}})
+
+    orderDetails.map(orderDetail => {
+      orderDetail.orderId = orderId
+      let menuPrice = menus.find(menu => menu.get('id') === orderDetail.menuId).get('price')
+      let modifiersPrice = getModifiersPrice(modifiers, orderDetail.modifierIds)
+      orderDetail.price = (parseFloat(menuPrice) + modifiersPrice) * orderDetail.quantity
+      orderDetail.modifierIds = orderDetail.modifierIds.toString()
+    })
+
+    return orderDetails
+  } catch (error) {
+    throw new Error(error.message)
+  }
+}
+function getModifiersPrice(modifiers, modifierIds) {
+  try {
+    let modifiersPrice = 0
+    modifierIds.forEach(modifierId => {
+      let modifierPrice = modifiers.find(modifier => modifier.get('id') === modifierId).get('price')
+      modifiersPrice += parseFloat(modifierPrice)
+    })
+    return modifiersPrice
+  } catch (error) {
+    throw new Error(error.message)
+  }
+}
+const resolvers = {
+  RootQuery: {},
+  RootMutation: {
+    async placeOrder(_, {input}, {loggedInUser}) {
+      _auth(loggedInUser)
+      try {
+        let orderId
+        return sequelize
+          .transaction(async t => {
+            await Order.create(input, {transaction: t}).then(async ({id}) => {
+              orderId = id              
+              await OrderDetail.bulkCreate(await createOrderDetail(input.orderDetails, id), {transaction: t})
+            })
+          })
+          .then(() => {
+            return orderId
```

Comment:
return createdOrder
(return the order is more usefull , plus no need to create a orderId to pass back to the API query